### DISCUSSION TITLE
Tesla not 'falling asleep' since recent update to 1.28.5

### DISCUSSION THREAD

**Discussion by keithdw1972** [teslamate-org/teslamate#3814] [Q&A] [answered]
Since a recent update... i think possibly the 1.28.5 upgrade, the car has not 'slept'.

Does anyone know where / which logs I can check to see what is keeping it awake?

**Comment by clarity-phev** [comment, marked as answer]
See this thread
https://github.com/teslamate-org/teslamate/discussions/3753
